pkg/kubernetes/chart_versions.go

```diff
@@ -0,0 +1,19 @@
+package kubernetes
+
+// This can be deprecated in the future (>= 1.0) when chart versions align to runtime versions.
+var chartVersionsMap = map[string]string{
+	"0.7.0": "0.4.0",
+	"0.7.1": "0.4.1",
+	"0.8.0": "0.4.2",
+	"0.9.0": "0.4.3",
```

Comment:
lets add 0.10.0 version here as its coming out next week, else it would need another change to cli.

Reply:
We plan to start versioning the chart versions to the Dapr runtime versions beginning 0.10.0

Reply:
sounds good!